python/nav/eventengine/plugins/upgrade.py

```diff
@@ -44,10 +44,8 @@ def _post_alert(self, event):
         ):
             alert.history_vars = {
                 "old_version": alert["old_version"],
-                "new_version": event.get_subject()
-                .entity_set.filter(software_revision__isnull=False)
-                .first()
-                .software_revision,
             }
+            if "new_version" in alert.keys():
+                alert.history_vars["new_version"] = alert["new_version"]
```

Comment:
Why does it need verify the presence of one variable, but not the other? The copy operation should be identical for both vars.. (and of course, verifying the existence of a key before accessing it is always a good idea, event for the `old_version`)

Reply:
That is true, I had a mistake in my thinking, I somehow assumed that it could be possible for the eventengine to get events that only have old version in the varmap, but not the new version. But now that I think about it further that is not gonna happen as long as we merge #2545 before this PR.

Reply:
But still, you never know if "something" will inject this type of event without those variables (they could be injected by a developer using the event generator scripts, or a third party script, or a code error created these events without the variables), so it's always a good idea to either verify it's presence in the dictionary - or use `get` on the dictionary with a default value of `N/A` (since various alert message templates may expect these vars to be present, and `N/A` would at least acknowledge that we "don't know").